app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api.v1::GreetingsController < ApplicationController
```

Comment:
- [ ] After that error I also go another one due to the missing module name. You need to change this line to this:

  `class Api::V1::GreetingsController < ApplicationController`

  ![api-v1](https://user-images.githubusercontent.com/81468675/183675319-eb786f31-573a-4095-a0c3-3fe24bdc6b48.jpg)